main.go

```diff
@@ -152,6 +157,7 @@ func main() {
 	}
 
 	r := mux.NewRouter()
+	r.HandleFunc("/healthcheck{_dummy:/?}", wrapper(HealthCheckHandler)).Methods("GET", "HEAD")
```

Comment:
I think it's best to set a path that starts with "_" because there are restrictions on the bucket name. 

> Bucket names must start and end with a number or letter.
https://cloud.google.com/storage/docs/buckets#naming

I also think it is better to set "/_health" as a simpler path.